.github/workflows/go_lint.yml

```
@@ -17,7 +17,7 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
-          version: v1.32
+          version: v1.49
```

Released yesterday :smile_cat: 
```suggestion
          version: v1.50
```

---

@tonyskapunk done.